scicomp/rsynconwindows.rst

```diff
@@ -0,0 +1,20 @@
+===========================
+Installing rsync on Windows
+===========================
```

Comment:
So I wonder how practical it is to recommend this (even if it is the theoretical best method).  I remember the day of trying to help people transfer stuff by scp/rsync before the age of SMB mounts.  This wasn't fun and I'm not sure how many people even were able to do it.  That's when I realized we needed SMB mounting to make things reasonably usable for most people.

Considering this, do you have any more thoughts on the role of this and is there some other thing to recommend as the top-level option?  (see also other note which I think is in the comment feed, but not here anymore because the file moved).